README.md

```diff
@@ -1 +1,203 @@
-# TDD-Project
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<a name="readme-top"></a>
+
+
+
+# 📖 OOP school library <a name="about-project"></a>
+
+
+**OOP school library**
+This project focused on building a Ruby school library app. Key achievements include:
+1- Implemented three classes: Person, Student, and Teacher.
+2- Utilized encapsulation and inheritance principles.
+3- Enabled command-line program execution.
+4- Followed Ruby best practices for code quality and documentation.
+
+This foundational work sets the stage for future app development and functionality expansion.
```

Comment:
- [ ] This project is supposed to have information about Test Driven Development not OOP, please kindly update this part of your code and every other place so that iy references the TDD project not OOP :+1: 
- [ ] Make sure to go through every section and update it accordingly for approval :+1: :sparkles: